scripts/generate_analysis_results.py

```diff
@@ -0,0 +1,142 @@
+import pandas as pd
+import os
+import seaborn as sns
+import datetime
+import chardet
+import time
+from pandas.api.types import is_number
+import matplotlib.pyplot as plt
+import numpy as np
+import itertools
+
+from matplotlib import pyplot as plt
+from fpdf import FPDF
+
+class ReportAnalysis:
+
+    path_ground_truth = ''
+    ground_truth_name = ''
+
+    def __init__(self, path_ground_truth, ground_truth_name):
+        self.path_ground_truth = path_ground_truth
+        self.ground_truth_name = ground_truth_name
+        self.generate_results()
+
+    @staticmethod
+    def generate_results():
+        FP = 0
+        TP = 0
+        FN = 0
+        TN = 0
+
+        print("Reading analyses execution results...")
+        soot_results = pd.read_csv('../miningframework/output/data/soot-results.csv', sep=';', encoding='latin-1', on_bad_lines='skip', low_memory=False)
+
+        loi = pd.read_csv(path_ground_truth, sep=';', encoding='latin-1', on_bad_lines='skip', low_memory=False)
+
+        print("Generating results...")
+
+        LOIGroundTruth = []
+        for LOI in loi[ground_truth_name]:
```

Comment:
@galilasmb ainda tem um problema aqui. o código assume que o i-essimo elemento de loi corresponde ao i-essimo elemento de soot-results, sem conferir que eles realmente se referem ao mesmo cenário de merge. isso é frágil. importante verificar. aí nem precisaria desse primeiro for aqui

Reply:
Isso mesmo, como entrada, teremos o arquivo de cenários com o Ground Truth correspondente à saida da execução do experimento, onde o i-ésimo elemento de cada arquito são correspondentes.